src/electionguard/elgamal.py

```diff
@@ -1,4 +1,4 @@
-from dataclasses import dataclass
+from pydantic.dataclasses import dataclass
```

Comment:
Linting is failing for legit reason. The imports are in wrong order. 

`pydantic` is not a built-in module like `dataclasses` or `typing`